src/nft/components/explore/Banner.tsx

```diff
@@ -77,6 +84,8 @@ const Banner = () => {
     }
   )
 
+  const collections = data?.filter((collection) => !EXCLUDED_COLLECTIONS.includes(collection.address)).slice(0, 5)
```

Comment:
done! I think I would argue whether there's a benefit of using useMemo in this context instead of filtering on a relatively small data set. However, I think it makes sense to keep the conde consistent (especially in context of this file), subject to any changes in the future!